src/main/org/apache/tools/ant/types/selectors/modifiedselector/ModifiedSelector.java

```diff
@@ -686,9 +686,7 @@ public void addParam(Parameter parameter) {
 
 
     /**
-     * Defined in org.apache.tools.ant.types.Parameterizable.
-     * Overwrite implementation in superclass because only special
-     * parameters are valid.
+     * Set all the Parameters for this custom selector, collected by\n the ExtendSelector class.
```

Comment:
there seems to be something wrong with the new comment (literal `\n`) and I don't understand what ExtendSelector refers to.